productcomments.php

```diff
@@ -713,7 +707,7 @@ public function renderCriterionForm($id_criterion = 0)
                         'lang' => true,
                         'label' => $this->trans('Criterion name', [], 'Modules.Productcomments.Admin'),
                         'name' => 'name',
-                        'desc' => $this->trans('Max length %s characters', [ProductCommentCriterion::NAME_MAX_LENGTH], 'Modules.Productcomments.Admin')
+                        'desc' => $this->trans('Max length %s characters', [ProductCommentCriterion::NAME_MAX_LENGTH], 'Modules.Productcomments.Admin'),
```

Comment:
What about _Maximum length: %s characters_ in order to avoid using too many abbreviations within the software?

Reply:
Also, just wondering: is it problematic to have two coding standards inside the same pull request? For instance, translation keys are here introduced by both `$this->trans` and `l s=` + `d=`.

Reply:
Yes it's ok `$this->trans` is the modern use (available in PHP)
In smarty we stil have to use `{l s="key" d="domain"}` but as you can see we use `domain` instead of `mod`, `domain` is the modern way

Reply:
Done

Reply:
Okay, nice, I wasn't sure that the two writings could mix. :-)
Watch out, it needs to have a punctuation sign between the two propositions, it should be _Maximum length: %s characters_ instead.

Reply:
Done